### ISSUE TITLE
Set up dev UAA to grant an application appropriate scope for CF API requests

### ISSUE THREAD

**Issue by jduss4** [cloud-gov/cg-ui#46]
Blocked by #60 

Similar to the process used to set up paas-admin, we need to register our application as a "client" that UAA recognizes, and then when logging in our application and users will be granted the appropriate scope to interact with the CF API.

I believe the steps will look something like this:

- [x] Learn about how client scopes vs user scopes work and what we need to get started
- [x] Open a PR with the [cg-deploy-cf](https://github.com/cloud-gov/cg-deploy-cf/blob/main/bosh/opsfiles/clients.yml#L134) repo to request a client be added
- [x] Deploy a version of our application and configure it to connect to UAA, verify we are receiving the expected scopes

Stretch goal / should we also check?

- [x] That a logged in user can send a basic query to the CF API with their token and receive a response?

Success criteria:

- [x] the development instance of UAA recognizes our application as a client and we have the id / secret needed to authenticate
- [x] our application can log in a user and receive back scopes we believe are necessary to interact with the CF API
